index.html

```diff
@@ -1441,7 +1441,7 @@ <h2>
         <p data-tests="onmerchantvalidation-attribute.https.html">
           A <a>PaymentRequest</a>'s <a>onmerchantvalidation</a> attribute is an
           <a>EventHandler</a> for a <a>MerchantValidationEvent</a> named
-          "<a>merchantvalidation</a>".
+          <a>merchantvalidation</a>.
```

Comment:
Going to keep these, as they map nicely to literal strings in c++.

Reply:
Can you elaborate on this? Who would reasonably make that connection and why are the other handful of events in this list not in the same situation?

Reply:
Ah, we should fix those to make sure they have quotes. 

This is what I mean about the string literals in c++: 
https://searchfox.org/mozilla-central/source/dom/payments/PaymentRequest.cpp#1001